app/generated_resources_override.grd

```diff
@@ -149,6 +149,9 @@ are declared in tools/grit/grit_rule.gni.
       <if expr="is_macosx">
         </if>
       <if expr="is_win or desktop_linux">
+        <message name="IDS_NEW_INCOGNITO_WINDOW" desc="The text label of a menu item for opening a new Private window">
```

Comment:
Sadly, this will not work. `_override.grd` files are auto-generated by running `npm run rebase_chromium_l10n`. The strings in the `_override` files are based on automatic generic substitutions that are mapped in https://github.com/brave/brave-core/blob/master/script/lib/grd_string_replacements.py
There is no mapping for `Incognito` -> `private` (maybe it's safe to add one if we adjust regex to make sure it's not the first word in the phrase? cc: @bsclifton). Barring that, I think the best option for this is to add a new string into `app/brave_generated_resources.grd` and then replace `IDS_NEW_INCOGNITO_WINDOW` with our string ID in code via a `chromium_src` override.

Reply:
Thanks for clarification @mkarolin 😄 I updated https://github.com/brave/brave-browser/wiki/Strings-and-Localization#chromium-strings to capture that detail (I had forgotten myself where the _override.grd comes from - just remembering I had asked here a while back https://github.com/brave/brave-browser/issues/17377#issuecomment-900031190)

I like the suggestion of using a `chromium_src` override (versus trying to hack regex). This unfortunately takes it out of the `good first issue` category

Reply:
I'll let you take it from here :)